install.sh

```diff
@@ -1,16 +1,24 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 
-mkdir -p $HOME/.local/share
-export PEARL_ROOT=$HOME/.local/share/pearl
-export PEARL_HOME=$HOME/.config/pearl
+[ -n "$1" ] && branch="--branch $1" || unset branch
```

Comment:
Can we use the double brackets ``[[`` ? I am trying to move all codebase with this.